nachomemes/bot.py

```diff
@@ -38,12 +40,21 @@ def mentioned_members(ctx: Context) -> Union[List[discord.Member], None]:
     "Returns the id of a memeber mentioned in a message."
     return [m for m in ctx.message.mentions]
 
+async def report(ctx: Context, e: Exception, message: str="An error has occured"):
+    """helper function to summarize or print the traceback of an error"""
+    err = traceback.format_exc()
+    # print(err, file=sys.stderr)
```

Comment:
I think we should always log to the console here, just in case. It may be useful in the future for pulling errors from our logs.

Reply:
we are re-throwing the error, so it is gonna end up in the console anyway.  printing it means it shows up twice.